dev/io.openliberty.org.jboss.weld6/src/com/ibm/ws/cdi/impl/weld/validation/LibertyDelegatingValidator.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *******************************************************************************/
+
+package com.ibm.ws.cdi.impl.weld.validation;
+
+import java.util.Collection;
+
+import org.jboss.weld.bootstrap.BeanDeployment;
+import org.jboss.weld.bootstrap.Validator;
+import org.jboss.weld.exceptions.DefinitionException;
+import org.jboss.weld.logging.MessageCallback;
+import org.jboss.weld.manager.BeanManagerImpl;
+
+import jakarta.enterprise.inject.spi.Bean;
+import jakarta.enterprise.inject.spi.Decorator;
+import jakarta.enterprise.inject.spi.InjectionPoint;
+import jakarta.enterprise.inject.spi.Interceptor;
+import jakarta.enterprise.inject.spi.Producer;
+
+public abstract class LibertyDelegatingValidator extends Validator {
```

Comment:
These need to be generated individually, not copy-pasted, so that we're sure we're generating the delegator class from the Validator class in that version of Weld.